exchange/bidder.go

```diff
@@ -74,6 +74,10 @@ const (
 	Gzip string = "GZIP"
 )
 
+var (
+	ErrTmaxAdjustment = errors.New("PBS cannot get response from bidder within time left for request hence timing out.")
```

Comment:
`ErrTmaxAdjustment` should be private. How about renaming error to `tmaxTimeoutErr`

```
tmaxTimeoutErr = errors.New("exceeded tmax duration")

```

Reply:
https://github.com/prebid/prebid-server/pull/2750/commits/d8cbdcb53c3b52a6de62769458259118e74c770d